listenbrainz/webserver/static/js/src/cb-review/CBReviewModal.tsx

```diff
@@ -546,7 +546,7 @@ export default class CBReviewModal extends React.Component<
               data-toggle="dropdown"
               type="button"
             >
-              {`${entityToReview.name} (
+              {`${entityToReview.name}(
               ${entityToReview.type.replace("_", " ")})`}
```

Comment:
I think this should work to remove the space before the entity type while keeping the space between the entity name and the parens:
```suggestion
              {`${entityToReview.name} (${entityToReview.type.replace(
                "_",
                " "
              )})`}
```